templates/index.js

```diff
@@ -0,0 +1,21 @@
+function formatCrateName(crateTitleAndVersion) {
+    const stringParts = crateTitleAndVersion.split(" ", 2);
+    return stringParts[0] + ' = "' + stringParts[1] + '"';
```

Comment:
I think this would be a good use case for destructuring, but it's not supported in IE 11 :(

Reply:
Yeah, it would have been _really_ nice to use a template literal, but a lot of the JS "nice-to-haves" aren't consistently supported without adding a build step :disappointed: